environment.yml

```diff
@@ -4,40 +4,46 @@ channels:
     - defaults
 dependencies:
     - python==3.9.7
-    - asdf
     - astropy
     - batman-package
     - bokeh[version='<3.0']
     - ccdproc
-    - celerite
+    - celerite # Needed for GP
     - corner
     - cython
-    - dynesty[version='>1.0']
-    - emcee[version='>3.0.0']
-    - george
-    - gwcs
-    - h5py[version='<3.2']
+    - dynesty[version='>1.0'] # Lower limit needed for specific arguments
+    - emcee[version='>3.0.0'] # Lower limit needed for specific arguments
+    - flake8 # Needed for testing
+    - george # Needed for GP
+    - h5py
+    - ipykernel
     - ipython
+    - jupyter
     - lmfit
     - matplotlib
     - nbsphinx # Needed for documentation
-    - numpy[version='>=1.20.0']
+    - numpy[version='>=1.20.0,<=1.23'] # Upper limit needed for Apple silicon
```

Comment:
I see that the upper limit is 1.23 here and 1.22 in [environment_pymc3.yml](https://github.com/kevin218/Eureka/pull/558/files#diff-15acc9e6ddc11f0b6781d4f3a13a02fe1aa14a2aeb776a107b611f63ab4a3c95).  What's the reason for this difference?  Would it just be better to remain consistent and adopt 1.22 here as well or should we aim for maximum flexibility on the version numbers?